radixdlt-core/radixdlt/src/main/java/com/radixdlt/keys/PersistedBFTKeyModule.java

```diff
@@ -51,6 +52,12 @@ ECPublicKey key(@Self BFTNode bftNode) {
 		return bftNode.getKey();
 	}
 
+	@Provides
+	@Self
+	ECKeyPair ecKeyPair(PersistedBFTKeyManager keyManager) {
```

Comment:
I don't think we should be exposing the private key in this way, better would be if exposed a Signer instead.

Reply:
E.g. the interface we would have if we were to use a private key in hardware